pennylane/_device.py

```diff
@@ -110,9 +110,7 @@ class Device(abc.ABC):
     """
 
     # pylint: disable=too-many-public-methods
-    _capabilities = {
-        "model": None,
-    }
+    _capabilities = {"model": None, "supports_tracker": True}
```

Comment:
For which devices will we have `"supports_tracker": False`? :thinking: If this entry is indeed added to `_capabilities`, then we'll need to make sure that it's being set correctly for all devices in PennyLane core and in the plugins on a case by case basis.

Reply:
I did have to update a bunch of device tests.

This also helps with compatibility with plugins.  If the device doesn't explicitly set `"supports_tracker"`, then it is assumed to be `False`.

Reply:
Is that so? I'd assume that setting here will propagate to most devices due to the inheritance logic with QubitDevice, unless explicitly set to False.

Reply:
So do you thinking the `"supports_tracker"` capability descriptor has no use and I should get rid of it?

Maybe I could instead use `if hasattr(dev, "tracker")` to check it's initialized with a tracker instance.

Reply:
> Maybe I could instead use if hasattr(dev, "tracker") to check it's initialized with a tracker instance.

Maybe this is a good idea.